docs/wip/exchange-onboarding.md

```diff
@@ -0,0 +1,352 @@
+
+    {
+        "status": "success",
+        "data": {
+            "id": "Ae2tdPwUPE...8V3AVTnqGZ",
+            "name": "MyFirstWallet",
+            "balance": 0
+        },
+        "meta": {
+            "pagination": {
+                "totalPages": 1,
+                "page": 1,
+                "perPage": 1,
+                "totalEntries": 1
+            }
+        }
+    }
+You have just created your first wallet. Information about this wallet can be retrieved using the GET /api/v1/wallets/{walletId} endpoint as follow:
+
+    curl -X GET https://localhost:8090/api/v1/wallets/{{walletId}} \
+         -H "Accept: application/json; charset=utf-8"              \
+         --cacert ./scripts/tls-files/ca.crt                       \
+
+## Receiving Money
+
+To receive money from other users you should provide your address. This address can be obtained from an account. Each wallet contains at least one account, you can think of account as a pocket inside of your wallet. Besides, you can view all existing accounts of a wallet by using the GET /api/v1/wallets/{{walletId}}/accounts endpoint as follow:
+
+    curl -X GET https://localhost:8090/api/v1/wallets/{{walletId}}/accounts?page=1&per_page=10 \
+         -H "Accept: application/json; charset=utf-8"                                          \
+         --cacert ./scripts/tls-files/ca.crt                                                   \
+Since you have, for now, only a single wallet, you’ll see something like this:
+
+    {
+        "status": "success",
+        "data": [
+            {
+                "index": 2147483648,
+                "addresses": [
+                    "DdzFFzCqrh...fXSru1pdFE"
+                ],
+                "amount": 0,
+                "name": "Initial account",
+                "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"
+            }
+        ],
+        "meta": {
+            "pagination": {
+                "totalPages": 1,
+                "page": 1,
+                "perPage": 10,
+                "totalEntries": 1
+            }
+        }
+    }
+
+Each account has at least one address, all listed under the addresses field. You can communicate one of these addresses to receive money on the associated account.
+
+## Sending Money
+
+In order to send money from one of your account to another address, you can create a new payment transaction using the POST /api/v1/transactions endpoint as follow:
+
+    curl -X POST https://localhost:8090/api/v1/transactions \
+         -H "Content-Type: application/json; charset=utf-8" \
+         -H "Accept: application/json; charset=utf-8"       \
+         --cacert ./scripts/tls-files/ca.crt                \
+         -d '{                                              \
+      "destinations": [{                                    \
+        "amount": 14,                                       \
```

Comment:
Indicate whether the units are Ada or Lovelace.

Reply:
In fact, if we can persuade the devs to make the units obvious in the key name that would be best.

cc @KtorZ

Reply:
I cannot remember on top of my head if `amount` is  a fractional number of an unsigned integer. If the latter than maybe you are right, labeling that `lovelaces` might be better, but I have to say I'm a bit on the fence as if people skipped the section or are not familiar with the distinction between ADA vs lovelace this can be even more confusing :)